src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -8,6 +8,7 @@ import cromwell.engine.backend.jes.Run.{Failed, Running, Success, _}
 import cromwell.engine.db.{JesStatus, JesId, JesCallBackendInfo, DataAccess}
 import cromwell.util.google.GoogleScopes
 import org.slf4j.LoggerFactory
+import DataAccess._
```

Comment:
If you rename the member in the DataAccess companion object to `dataAccess`, or  

```scala
import DataAccess.{slickDataAccess => dataAccess}
```

that would minimize the changes, and also make clear you're coding to the `DataAccess` trait and nothing particular about `SlickDataAccess`.  I'd suggest the former, since currently the member is named `slickDataAccess` but typed `DataAccess`.